Source/Scene/Scene.js

```diff
@@ -3332,8 +3351,13 @@ function updateAndClearFramebuffers(scene, passState, clearColor) {
 
   const passes = scene._frameState.passes;
   const picking = passes.pick;
+  if (defined(view.globeDepth)) {
+    view.globeDepth.picking = picking;
+  }
   const useWebVR = environmentState.useWebVR;
 
+  const originalSamples = scene.numberSamples;
```

Comment:
Is `originalSamples` needed? Unless I missed it I don't see anything changing the scene's `numberSamples`.